src/components/transactions/TransactionList.vue

```diff
@@ -31,6 +69,48 @@ export default {
       type: Object,
       required: true
     }
+  },
+  computed: {
+    groupedTransactions() {
+      return orderBy(
+        groupBy(this.categorizedTransactions, "title"),
+        group => group[0].tx.time,
+        "desc"
+      )
+    },
+    categorizedTransactions() {
+      return this.transactions.map(tx => {
+        // check if the tx is in Today, Yesterday or Last Week
+        const category = categories.find(({ matcher }) => matcher(tx))
+        if (category) {
+          return {
+            title: category.title,
+            tx
+          }
+        }
+
+        // check if tx is in a month this year
+        const date = moment(tx.time)
+        const today = moment()
+        if (date.year() === today.year()) {
+          return {
+            title: date.format("MMMM"),
```

Comment:
any reason we're grouping by month instead of day?

Reply:
See Colins proposal in the issue.

Reply:
ah didn't see that.

@colw what's your thinking here?

i'd like to remove date from the tx itself which won't be possible if we group by something other than date (day).

Reply:
If there is a 1:1 between date-title and transaction, it can add to the clutter, and defeat the purpose of a heading/separator. Also, as time goes on, the exact date becomes less important, hence grouping by month. At the top, or present time, it matters what day transactions took place.

Also, perhaps easier to read when formatted relatively. For example, I might not remember the exact date off-hand, but I certainly know when 'yesterday' was.

I'm quite flexible, I think we have the same goal of reducing clutter without hiding anything too important.

Reply:
interesting line of thinking! thanks for sharing. i suppose this highlights the value of user stories (which we have not written down / agreed upon).

in my mind the point stands though - if we are not grouping by date, then we can't remove date from the transaction component. the new groupings struck me as 1) new and 2) additional abstractions.

Reply:
>  if we are not grouping by date, then we can't remove date from the transaction component.

Of course. I wasn't aware of the direction when making the above suggestion. Your call, I like your approach too.